<html>
<head>
<title>TestNG:  Default test(failed)</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Default test(failed)</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>0/9/0</td>
</tr><tr>
<td>Started on:</td><td>Fri May 31 09:24:33 EAT 2019</td>
</tr>
<tr><td>Total time:</td><td>32 seconds (32707 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='TestCases.LoginDataProvider.userLogin()'><b>userLogin</b><br>Test class: TestCases.LoginDataProvider<br>Parameters: null, null<br/>
<a href="#Output-1479909053" onClick='toggleBox("Output-1479909053", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-1479909053" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-1479909053">
Failed Test Case : userLogin(null,null)<br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: expected [http://devops-deve.kifiya.et:8082/mfs-admin/dashboard?loggedInUserName=mfs&amp;appicationVersion=Build+Version%3A+1.1.0] but found [http://devops-deve.kifiya.et:8082/mfs-admin/]
	at TestCases.LoginDataProvider.userLogin(LoginDataProvider.java:57)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1479909053", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1479909053'><pre>java.lang.AssertionError: expected [http://devops-deve.kifiya.et:8082/mfs-admin/dashboard?loggedInUserName=mfs&amp;appicationVersion=Build+Version%3A+1.1.0] but found [http://devops-deve.kifiya.et:8082/mfs-admin/]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at org.testng.Assert.assertEquals(Assert.java:463)
	at TestCases.LoginDataProvider.userLogin(LoginDataProvider.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>3</td>
<td>TestCases.LoginDataProvider@6f195bc3</td></tr>
<tr>
<td title='TestCases.LoginDataProvider.userLogin()'><b>userLogin</b><br>Test class: TestCases.LoginDataProvider<br>Parameters: , <br/>
<a href="#Output-1692073709" onClick='toggleBox("Output-1692073709", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-1692073709" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-1692073709">
Failed Test Case : userLogin(,)<br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: expected [http://devops-deve.kifiya.et:8082/mfs-admin/dashboard?loggedInUserName=mfs&amp;appicationVersion=Build+Version%3A+1.1.0] but found [http://devops-deve.kifiya.et:8082/mfs-admin/]
	at TestCases.LoginDataProvider.userLogin(LoginDataProvider.java:57)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1692073709", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1692073709'><pre>java.lang.AssertionError: expected [http://devops-deve.kifiya.et:8082/mfs-admin/dashboard?loggedInUserName=mfs&amp;appicationVersion=Build+Version%3A+1.1.0] but found [http://devops-deve.kifiya.et:8082/mfs-admin/]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at org.testng.Assert.assertEquals(Assert.java:463)
	at TestCases.LoginDataProvider.userLogin(LoginDataProvider.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>3</td>
<td>TestCases.LoginDataProvider@6f195bc3</td></tr>
<tr>
<td title='TestCases.LoginDataProvider.userLogin()'><b>userLogin</b><br>Test class: TestCases.LoginDataProvider<br>Parameters: username, <br/>
<a href="#Output-638695843" onClick='toggleBox("Output-638695843", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-638695843" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-638695843">
Failed Test Case : userLogin(username,)<br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: expected [http://devops-deve.kifiya.et:8082/mfs-admin/dashboard?loggedInUserName=mfs&amp;appicationVersion=Build+Version%3A+1.1.0] but found [http://devops-deve.kifiya.et:8082/mfs-admin/]
	at TestCases.LoginDataProvider.userLogin(LoginDataProvider.java:57)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace638695843", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace638695843'><pre>java.lang.AssertionError: expected [http://devops-deve.kifiya.et:8082/mfs-admin/dashboard?loggedInUserName=mfs&amp;appicationVersion=Build+Version%3A+1.1.0] but found [http://devops-deve.kifiya.et:8082/mfs-admin/]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at org.testng.Assert.assertEquals(Assert.java:463)
	at TestCases.LoginDataProvider.userLogin(LoginDataProvider.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>2</td>
<td>TestCases.LoginDataProvider@6f195bc3</td></tr>
<tr>
<td title='TestCases.LoginDataProvider.userLogin()'><b>userLogin</b><br>Test class: TestCases.LoginDataProvider<br>Parameters: , password<br/>
<a href="#Output-864248990" onClick='toggleBox("Output-864248990", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-864248990" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-864248990">
Failed Test Case : userLogin(,password)<br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: expected [http://devops-deve.kifiya.et:8082/mfs-admin/dashboard?loggedInUserName=mfs&amp;appicationVersion=Build+Version%3A+1.1.0] but found [http://devops-deve.kifiya.et:8082/mfs-admin/]
	at TestCases.LoginDataProvider.userLogin(LoginDataProvider.java:57)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace864248990", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace864248990'><pre>java.lang.AssertionError: expected [http://devops-deve.kifiya.et:8082/mfs-admin/dashboard?loggedInUserName=mfs&amp;appicationVersion=Build+Version%3A+1.1.0] but found [http://devops-deve.kifiya.et:8082/mfs-admin/]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at org.testng.Assert.assertEquals(Assert.java:463)
	at TestCases.LoginDataProvider.userLogin(LoginDataProvider.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>2</td>
<td>TestCases.LoginDataProvider@6f195bc3</td></tr>
<tr>
<td title='TestCases.LoginDataProvider.userLogin()'><b>userLogin</b><br>Test class: TestCases.LoginDataProvider<br>Parameters: username, password<br/>
<a href="#Output-1649847375" onClick='toggleBox("Output-1649847375", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-1649847375" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-1649847375">
Failed Test Case : userLogin(username,password)<br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: expected [http://devops-deve.kifiya.et:8082/mfs-admin/dashboard?loggedInUserName=mfs&amp;appicationVersion=Build+Version%3A+1.1.0] but found [http://devops-deve.kifiya.et:8082/mfs-admin/authenticate]
	at TestCases.LoginDataProvider.userLogin(LoginDataProvider.java:57)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1649847375", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1649847375'><pre>java.lang.AssertionError: expected [http://devops-deve.kifiya.et:8082/mfs-admin/dashboard?loggedInUserName=mfs&amp;appicationVersion=Build+Version%3A+1.1.0] but found [http://devops-deve.kifiya.et:8082/mfs-admin/authenticate]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at org.testng.Assert.assertEquals(Assert.java:463)
	at TestCases.LoginDataProvider.userLogin(LoginDataProvider.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>2</td>
<td>TestCases.LoginDataProvider@6f195bc3</td></tr>
<tr>
<td title='TestCases.LoginDataProvider.userLogin()'><b>userLogin</b><br>Test class: TestCases.LoginDataProvider<br>Parameters: mfs, password<br/>
<a href="#Output-859123506" onClick='toggleBox("Output-859123506", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-859123506" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-859123506">
Failed Test Case : userLogin(mfs,password)<br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: expected [http://devops-deve.kifiya.et:8082/mfs-admin/dashboard?loggedInUserName=mfs&amp;appicationVersion=Build+Version%3A+1.1.0] but found [http://devops-deve.kifiya.et:8082/mfs-admin/authenticate]
	at TestCases.LoginDataProvider.userLogin(LoginDataProvider.java:57)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace859123506", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace859123506'><pre>java.lang.AssertionError: expected [http://devops-deve.kifiya.et:8082/mfs-admin/dashboard?loggedInUserName=mfs&amp;appicationVersion=Build+Version%3A+1.1.0] but found [http://devops-deve.kifiya.et:8082/mfs-admin/authenticate]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at org.testng.Assert.assertEquals(Assert.java:463)
	at TestCases.LoginDataProvider.userLogin(LoginDataProvider.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>2</td>
<td>TestCases.LoginDataProvider@6f195bc3</td></tr>
<tr>
<td title='TestCases.LoginDataProvider.userLogin()'><b>userLogin</b><br>Test class: TestCases.LoginDataProvider<br>Parameters: username, system<br/>
<a href="#Output-1999664216" onClick='toggleBox("Output-1999664216", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-1999664216" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-1999664216">
Failed Test Case : userLogin(username,system)<br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: expected [http://devops-deve.kifiya.et:8082/mfs-admin/dashboard?loggedInUserName=mfs&amp;appicationVersion=Build+Version%3A+1.1.0] but found [http://devops-deve.kifiya.et:8082/mfs-admin/authenticate]
	at TestCases.LoginDataProvider.userLogin(LoginDataProvider.java:57)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1999664216", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1999664216'><pre>java.lang.AssertionError: expected [http://devops-deve.kifiya.et:8082/mfs-admin/dashboard?loggedInUserName=mfs&amp;appicationVersion=Build+Version%3A+1.1.0] but found [http://devops-deve.kifiya.et:8082/mfs-admin/authenticate]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at org.testng.Assert.assertEquals(Assert.java:463)
	at TestCases.LoginDataProvider.userLogin(LoginDataProvider.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>2</td>
<td>TestCases.LoginDataProvider@6f195bc3</td></tr>
<tr>
<td title='TestCases.LoginDataProvider.userLogin()'><b>userLogin</b><br>Test class: TestCases.LoginDataProvider<br>Parameters: ^&^&^&, system<br/>
<a href="#Output-820878444" onClick='toggleBox("Output-820878444", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-820878444" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-820878444">
Failed Test Case : userLogin(^&^&^&,system)<br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: expected [http://devops-deve.kifiya.et:8082/mfs-admin/dashboard?loggedInUserName=mfs&amp;appicationVersion=Build+Version%3A+1.1.0] but found [http://devops-deve.kifiya.et:8082/mfs-admin/authenticate]
	at TestCases.LoginDataProvider.userLogin(LoginDataProvider.java:57)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace820878444", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace820878444'><pre>java.lang.AssertionError: expected [http://devops-deve.kifiya.et:8082/mfs-admin/dashboard?loggedInUserName=mfs&amp;appicationVersion=Build+Version%3A+1.1.0] but found [http://devops-deve.kifiya.et:8082/mfs-admin/authenticate]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at org.testng.Assert.assertEquals(Assert.java:463)
	at TestCases.LoginDataProvider.userLogin(LoginDataProvider.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>2</td>
<td>TestCases.LoginDataProvider@6f195bc3</td></tr>
<tr>
<td title='TestCases.LoginDataProvider.userLogin()'><b>userLogin</b><br>Test class: TestCases.LoginDataProvider<br>Parameters: mfs, (*&&****<br/>
<a href="#Output-237297104" onClick='toggleBox("Output-237297104", this, "Show output", "Hide output");'>Show output</a>

<a href="#Output-237297104" onClick="toggleAllBoxes();">Show all outputs</a>
<div class='log' id="Output-237297104">
Failed Test Case : userLogin(mfs,(*&&****)<br/>
</div>
</td>
<td><div><pre>java.lang.AssertionError: expected [http://devops-deve.kifiya.et:8082/mfs-admin/dashboard?loggedInUserName=mfs&amp;appicationVersion=Build+Version%3A+1.1.0] but found [http://devops-deve.kifiya.et:8082/mfs-admin/authenticate]
	at TestCases.LoginDataProvider.userLogin(LoginDataProvider.java:57)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace237297104", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace237297104'><pre>java.lang.AssertionError: expected [http://devops-deve.kifiya.et:8082/mfs-admin/dashboard?loggedInUserName=mfs&amp;appicationVersion=Build+Version%3A+1.1.0] but found [http://devops-deve.kifiya.et:8082/mfs-admin/authenticate]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:453)
	at org.testng.Assert.assertEquals(Assert.java:463)
	at TestCases.LoginDataProvider.userLogin(LoginDataProvider.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>2</td>
<td>TestCases.LoginDataProvider@6f195bc3</td></tr>
</table><p>
</body>
</html>